lib/opentelemetry/wrapper.ts

```diff
@@ -118,7 +118,35 @@ const APICallBasedService = {
 	Firecrawl: ExternalServiceName.Firecrawl,
 } as const;
 
-type VercelBlobOperationType = "put" | "fetch";
+export const VercelBlobOperation = {
+	Put: {
+		type: "put" as const,
+		measure: (result: { size: number }) => ({
+			blobSizeStored: result.size,
+		}),
+	},
+	Fetch: {
+		type: "fetch" as const,
+		measure: (result: { size: number }) => ({
+			blobSizeTransfered: result.size,
+		}),
+	},
+	Del: {
+		type: "del" as const,
+		measure: (result: { size: number }) => ({
+			blobSizeStored: -result.size,
+		}),
+	},
+	List: {
+		type: "list" as const,
+		measure: (result: { size: number }) => ({
+			blobSizeTransfered: result.size,
```

Comment:
- `list()` is related to the pricing model with data transfer
  - ref: https://vercel.com/docs/storage/vercel-blob/usage-and-pricing#pricing